Cargo.toml

```diff
@@ -114,6 +114,8 @@ rug = { version = "1.18" }
 secp256k1 = { version = "0.28.0", default-features = false, features = ["recovery"] }
 serde = { version = "1.0.144", default-features = false, features = ["derive"] }
 serde_json = { version = "1.0.105" }
+serde_with = { version = "3.9.0", features = ["hex"] }
```

Comment:
This pulls in _lots_ of new transitive dependencies in the Cargo.lock.

Can we use `default-features = false` and only pull in the features we need? e.g. `serde_with` probably doesn't need `chrono_0_4`, `chrono`, or `time_0_3` features: https://docs.rs/serde_with/3.9.0/serde_with/#modules

Reply:
Good point. I will disable the default features.